File: client/model/response.py
class Response:
    def __init__(self, response_body: map) -> None:
        self.response_body = response_body

    def get_response_body(self) -> map:
        return self.response_body
    
    def set_response_body(self, response_body: map) -> None:
        self.response_body = response_body

    def print_response(self) -> None:
        print('\nResponse from server')
        for k, v in self.response_body.items():
            print(f'{k}: {v}')

File: server/src/model/Response.java
package model;

import java.util.HashMap;

public class Response {
    private HashMap<String, Object> responseBody;

    public Response(HashMap<String, Object> responseBody) {
        this.responseBody = responseBody;
    }

    public HashMap<String, Object> getResponseBody() {
        return responseBody;
    }

    public void setResponseBody(HashMap<String, Object> responseBody) {
        this.responseBody = responseBody;
    }

    public void addResponseBody(String key, Object value) {
        responseBody.put(key, value);
    }

    public Object getResponseBody(String key) {
        return responseBody.get(key);
    }
}
